src/lexers/javascript-lexer.js

```diff
@@ -1,39 +1,57 @@
-import * as acorn from 'acorn-jsx'
-import injectAcornObjectRestSpread from "acorn-object-rest-spread/inject"
+import * as acorn from 'acorn'
+import injectAcornStage3 from "acorn-stage3/inject"
 import injectAcornEs7 from "acorn-es7"
 import * as walk from 'acorn/dist/walk'
 import BaseLexer from './base-lexer'
 
+const WalkerBase = Object.assign({}, walk.base, {
+  Import(node, st, c) {
+    // We need this catch, but we don't need the catch to do anything.
+  }
+})
+
 export default class JavascriptLexer extends BaseLexer {
   constructor(options = {}) {
     super(options)
 
-    this.acornOptions = { sourceType: 'module', ...options.acorn }
+    this.acornOptions = { 
+      sourceType: 'module', 
+      ecmaVersion: 9,
+      ...options.acorn,
+      plugins: {
+        stage3: true,
+        es7: true,
+        ...(options.acorn ? options.acorn.plugins : {})
+      }
+    }
```

Comment:
These are my opinions -- ecmaVersion 9 supports the most JS, and defaulting stage3 and es7 means that the user has to worry about less config right out the gate. I understand that this may be a contentious change.

Reply:
I agree with your decision, I would make sure it is documented though

Reply:
Yeah! I was thinking about changing the table-based documentation to snippet based. For example, from this...

Option                   | Description                                           | Default
---------------------- | ----------------------------------------------------- | ---
**contextSeparator**     | Key separator used in your translation keys           | `_`
**createOldCatalogs**    | Save the \_old files                                  | `true`
... | ... | ...

... to this ...

```js
{
  "contextSeparator": "_",
  // Key separator used in your translation keys

  "createOldCatalogs": true,
  // Save the \_old files

  ...
}
```

Thoughts? It lets give them a full config of all defaults as well as letting us define nested config a little easier and without the markdown table restriction of a single line per cell.